docs/add-new-resource-short.md

```diff
@@ -395,7 +395,7 @@ detailed guide could also help you.
 [provider-guide]:
     https://github.com/upbound/upjet/blob/main/docs/generating-a-provider.md
 [config-guide]:
-    https://github.com/upbound/upjet/blob/main/docs/configuring-a-resource.md
+    https://github.com/upbound/upjet/blob/main/docs/add-new-resource-long.md
 [issue-258]: https://github.com/upbound/official-providers/issues/258
```

Comment:
This link is also broken but I cannot solve it.

Reply:
Similarly there is another one pointing to https://github.com/upbound/official-providers/issues/new/choose

Reply:
We may use the following link to solve this problem: `https://github.com/upbound/provider-aws/issues/90`